iped-parsers/iped-parsers-impl/src/main/java/dpf/ap/gpinf/telegramextractor/MapTypeMSG.java

```diff
@@ -41,7 +41,7 @@
 
         msg.put(6, Messages.getString("TelegramReport.pinnMessage"));
         msg.put(7, Messages.getString("TelegramReport.UserJoinLink"));
-        msg.put(8, Messages.getString("TelegramReport.ChangeToGroup"));
+        msg.put(8, Messages.getString("TelegramReport.ChannelMigratedFromGroup"));
```

Comment:
Thanks @hauck-jvsh! One question: are these mapping changes correct for old telegram versions, right?

Reply:
Yes, I will take another look just to be sure and put the reference. I think that it was wrong in the past.

Reply:
This the reference
```
 case 8:
      self = .channelMigratedFromGroup(title: decoder.decodeStringForKey("title", orElse: ""), groupId: PeerId(decoder.decodeInt64ForKey("groupId", orElse: 0)))
```
The file can be found in:
https://github.com/TelegramMessenger/Telegram-iOS/blob/master/submodules/TelegramCore/Sources/SyncCore/SyncCore_TelegramMediaAction.swift

Reply:
Thanks for double checking @hauck-jvsh